src/app/app-modules/nurse-doctor/sms-notification/sms-notification.component.css

```
@@ -0,0 +1,133 @@
+/* make the checkbox occupy same vertical space so baseline aligns */
+.sms-section .mat-checkbox {
+  display: flex;
+  align-items: center;
+  height: 56px; /* match material form-field height for outline appearance */
+  margin: 0;    /* remove extra margins */
+}
+
+/* constrain the phone field width and remove extra margin */
+.sms-section .alt-phone-field {
+  margin-top: 10px;
+  margin: 0;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)